core/commonTest/src/kotlinx/serialization/SealedDiamondTest.kt

```diff
@@ -0,0 +1,27 @@
+package kotlinx.serialization
+
+import kotlinx.serialization.descriptors.*
+import kotlin.test.*
+
+class SealedDiamondTest {
+
+    @Serializable
+    sealed interface A {}
+
+    @Serializable
+    sealed interface B: A {}
```

Comment:
Our code-style implies whitespace prior to `:`

Reply:
Apparently reformatter doesn't do that for some reason